elements/bulbs-dfp/bulbs-dfp.js

```diff
@@ -44,13 +42,23 @@ export default class BulbsDfp extends BulbsHTMLElement {
       },
     });
 
-    let intervalLength = parseFloat(this.getAttribute('refresh-interval', 10));
-    this.refreshInterval = window.setInterval(this.handleInterval, intervalLength);
+    let defaultRefreshInterval = 30000;
+    this.adUnitData = this.adsManager.units[this.dataset.adUnit];
+    if (!this.adUnitData.refreshDisabled) {
+
+      this.refreshInterval = window.setInterval(
+        this.handleInterval,
+        this.adUnitData.refreshInterval || defaultRefreshInterval
+      );
+    }
```

Comment:
The main change, will allow current ad refresh behavior if ad unit configuration doesn't have `refreshDisabled === true`. Interval can be changed via ad unit config `refreshInterval` value; if not set, refresh defaults to 30 seconds.